hack/generated/pkg/testcommon/test_context.go

```diff
@@ -75,9 +75,19 @@ func (tc TestContext) ForTest(t *testing.T) (PerTestContext, error) {
 		return PerTestContext{}, errors.Wrapf(err, "creating ARM client")
 	}
 
+	// As of https://github.com/Azure/go-autorest/releases/tag/autorest%2Fv0.11.9, the autorest
+	// client reuses HTTP clients among instances. We add handlers to the HTTP client based on
+	// the specific test in question, which means that clients cannot be reused.
+	// We explicitly create a new http.Client so that one tests recordings don't
```

Comment:
#nitpick #innerpedant
```suggestion
	// We explicitly create a new http.Client so that the recording from one test doesn't
```

Reply:
Fixed